src/effects/Equalization.cpp

```diff
@@ -449,11 +459,16 @@ struct EqualizationTask {
    float *thisWindow{ window1.get() };
    float *lastWindow{ window2.get() };
 
+   const WaveTrack &input;
+   Poller &poller;
+
    // create a NEW WaveTrack to hold all of the output,
    // including 'tails' each end
    std::shared_ptr<WaveTrack> output;
 
+   size_t M;
```

Comment:
I understand that you are restructuring the old code, but I think that a comment would be very welcome here

Reply:
I meant a comment in the source code, not on GitHub